### DISCUSSION TITLE
Welcome to JSBSim Discussions!

### DISCUSSION THREAD

**Discussion by bcoconni** [JSBSim-Team/jsbsim#353] [General]
## 👋 Welcome!
  We’re now using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bcoconni** [comment]
@JSBSim-Team I have just enabled the new GitHub feature "Discussions" for the JSBSim project. Discussions is basically a forum which allows interactions on any topic related to the project (including but not limited to answering users questions).

You are welcome to participate and add material to the forum/Discussions. From my point of view, the idea would be stopping using Issues  from now on for general questions and support requests and rather start using Discussions instead for such topics. From that perspective, Issues would be limited to bug reports and feature requests. Any other topic should go to Discussions.

Let me know what you think.

**Comment by jonsberndt** [comment]
I think this is a good idea using GitHub Discussions. 

And I'm looking forward to getting familiar with JSBSim. I hear it's pretty good! ;-)

**Comment by bcoconni** [comment]
> And I'm looking forward to getting familiar with JSBSim. I hear it's pretty good! ;-)

🤣 Thanks ! May be you could introduce yourself ? 😄

**Comment by seanmcleod** [comment]
Good idea. People will invariably post in the wrong place (Discussions versus Issues) but I see there is a **Convert to discussion** option in Issues which you've tested. I don't see the equivalent for the opposite direction yet.

**Reply by bcoconni** [reply]
> [...] but I see there is a Convert to discussion option in Issues which you've tested. I don't see the equivalent for the opposite direction yet.

Well if you click on the 3 dots on the upper right corner of a comment in Discussions there is an option "Reference in new issue". That might very well do the trick (no tested yet 😄)
